### ISSUE TITLE
Experimental effect

### ISSUE THREAD

**Issue by yinruonan** [ZitongYu/STVEN_rPPGNet#8]
Can the author explain the processing of the input data? I trained rPPGNet separately as described in the paper, but the effect is not very good. The output rPPG signal of the model is almost a straight line. In the experiment, I used the following steps to process the data:
1. For a single input sample, 64 frames are randomly sampled in the video. The dimension of each video segment is [3, T=64, 128, 128]. After noise reduction, the ECG signal samples 64 pieces of data at the corresponding time. This piece of training data is normalized to make it obey a normal distribution.
2. For the face mask, use bob.ip.skincolorfilter to set the threshold to 0.3 to get a [T=64,64,64] tensor with values ​​0 and 1.
Can someone help me please?
The training loss of each sample is between 3-4, and there is no downward trend.

**Comment by ZitongYu** [comment]
Hi,

I suggest you train on COHFACE or UBFC-RPPG dataset first as they have the
BVP ground-truth, which is easier to converge.

Thx.
--Zitong


yinruonan <notifications@github.com> 于2020年12月9日周三 上午6:41写道：

**Comment by yinruonan** [comment]
Thank you!  I have got the results of the experiment.  When processing data, a function was written incorrectly, so that the result is always a straight line.

**Comment by Biinzy** [comment]
Can you please post the code for preprocessing the data?
